rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/SmeupInterpreterTest.kt

```diff
@@ -119,4 +119,10 @@ open class SmeupInterpreterTest : AbstractTest() {
         val expected = listOf("999-9999", "A90_A4(        ) A90_A5(RPG DEPT)", "A90_A4(        ) A90_A5(RPG DEPT)")
         assertEquals(expected, "smeup/T10_A90".outputOf())
     }
+
+    @Test
+    fun executeA02_P12() {
```

Comment:
Wrong name, the naming convention is `Txx_Ayy` with `xx` program id and `yy` section id

Reply:
renamed